src/Athena.Importer/data/requirement/eecs-electric-circuits-i.json

```diff
@@ -0,0 +1,5 @@
+{
+  "id": "890c78c1-5b8c-448c-8b08-57a78ee37504",
```

Comment:
Multiple requirements with this ID exist:

```powershell
id                                   name
--                                   ----
890c78c1-5b8c-448c-8b08-57a78ee37504 COLLEGE PHYSICS I
890c78c1-5b8c-448c-8b08-57a78ee37504 COLLEGE PHYSICS II
890c78c1-5b8c-448c-8b08-57a78ee37504 EECS Data Communications
890c78c1-5b8c-448c-8b08-57a78ee37504 EECS Electric Circuits I
890c78c1-5b8c-448c-8b08-57a78ee37504 EECS First Year Design 
```

Reply:
done